src/main/java/io/cdap/plugin/servicenow/source/ServiceNowRecordReader.java

```diff
@@ -185,27 +188,27 @@ private Object convertToValue(String fieldName, Schema fieldSchema, Map<String,
     }
   }
 
-  private String convertToStringValue(Object fieldValue) {
+  public String convertToStringValue(Object fieldValue) {
```

Comment:
can we make these methods package private?

Reply:
No for now as they are being used directly in the test class ServiceNowRecordReaderTest.

I will try writing a test case that does not directly call these methods but still covers them in this [PR](https://github.com/data-integrations/servicenow-plugins/pull/9)

Reply:
Yes, Changed it to package-private and added the annotation: ` @VisibleForTesting` 69415df